internal/host/hostcommands/instruction_manager.go

```diff
@@ -31,8 +31,9 @@ const (
 )
 
 type StepsStruct struct {
-	Commands      []CommandGetter
-	NextStepInSec int64
+	Commands       []CommandGetter
+	NextStepInSec  int64
+	PostStepAction string
```

Comment:
What is the usage of this field ?

Reply:
This field is used to tell the agent what to do after it finished executing the current bunch of the command (steps) sent to it. By default , we send Continue action, which tells agents to continue processing (sleep and then send get next step command. In case of empty command we send the agent action of exit, which causes it to exit the current loop and to initiate registration to the AI again